lib/improver/tests/ensemble_calibration/ensemble_calibration/test_ApplyCoefficientsForEnsembleCalibration.py

```diff
@@ -271,245 +275,169 @@ def test_unmatching_y_axis(self):
+        location_parameter = (
+            self.plugin._calculate_location_parameter_from_realizations(
+                optimised_coeffs))
         self.assertCalibratedVariablesAlmostEqual(
-            forecast_predictors.data, expected_forecast_predictors.data)
+            location_parameter,
+            self.expected_loc_param_no_statsmodels_realizations)
+        assert_array_almost_equal(
+            location_parameter,
+            self.expected_loc_param_mean,
+            decimal=0)
+        assert_array_almost_equal(
+            location_parameter,
+            self.expected_loc_param_statsmodels_realizations,
+            decimal=0)
 
 
-class Test_calibrate_forecast_data(
+class Test__calculate_scale_parameter(
         SetupCoefficientsCubes, EnsembleCalibrationAssertions):
 
-    """Test the calibrate_forecast_data method.
-
-    Note that there are several cross comparisons of results between the tests.
-    These overlap to ensure that if any one test is removed, the others still
-    check that the behaviour is as expected."""
+    """Test the _calculate_scale_parameter method."""
 
-    @ManageWarnings(
-        ignored_messages=["Collapsing a non-contiguous coordinate."])
     def setUp(self):
-        """Setup cubes and sundries for testing calibration."""
+        """Set-up the plugin for testing."""
         super().setUp()
-
-        self.forecast_vars = self.current_temperature_forecast_cube.collapsed(
-            "realization", iris.analysis.VARIANCE)
-        self.forecast_predictor = (
-            self.current_temperature_forecast_cube.collapsed(
-                "realization", iris.analysis.MEAN))
-
         self.plugin = Plugin()
+        self.plugin.current_forecast = self.current_temperature_forecast_cube
 
-    def test_mean_as_predictor(self):
-        """Test that the calibrated forecast using ensemble mean as the
-        predictor returns the correct values. Check that the calibrated mean
-        is similar to when the ensemble realizations are used as the predictor
-        with and without statsmodels."""
-
+    @ManageWarnings(
+        ignored_messages=["Collapsing a non-contiguous coordinate."])
+    def test_basic(self):
+        """Test the scale parameter is calculated correctly."""
         optimised_coeffs = dict(
             zip(self.coeffs_from_mean.coord("coefficient_name").points,
                 self.coeffs_from_mean.data))
-        predicted_mean = self.expected_calibrated_predictor_mean.flatten()
-
-        calibrated_forecast_predictor, calibrated_forecast_var = (
-            self.plugin.calibrate_forecast_data(
-                optimised_coeffs, predicted_mean, self.forecast_predictor,
-                self.forecast_vars))
-
-        self.assertCalibratedVariablesAlmostEqual(
-            calibrated_forecast_predictor.data,
-            self.expected_calibrated_predictor_mean)
+        scale_parameter = (
+            self.plugin._calculate_scale_parameter(optimised_coeffs))
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Access to a protected member _calculate_scale_parameter of a client class](https://app.codacy.com/manual/metoppv_tech/improver/pullRequest?prid=4714944)